.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
   - repo: https://github.com/PyCQA/isort
-    rev: 5.10.1
+    rev: 5.12.0
```

Comment:
Let me check if we can remov this line since we basically manage versions in https://github.com/appium/python-client/blob/b142e00e3f47065752c80f71f2f8cb80bf2500f7/Pipfile#L9 . I guess this version can be removed. (but it is not this pr)